react-javascript/browserslist

```diff
@@ -0,0 +1,31 @@
+# This is an rcfile for Browserslist (https://www.npmjs.com/package/browserslist)
```

Comment:
Couldn't we use a shareable config for this file?

Reply:
We could try to extract it if it would be useful to the other language/framework examples. In this case it's here because the React project generator includes it. I don't know what the best way to share it across the projects would be?